selfdrive/car/toyota/interface.py

```diff
@@ -173,9 +173,9 @@ def get_params(candidate, fingerprint, vin="", is_panda_black=False):
       ret.wheelbase = 2.68986
       ret.steerRatio = 14.3
       tire_stiffness_factor = 0.7933
-      ret.lateralTuning.pid.kpV, ret.lateralTuning.pid.kiV = [[0.6], [0.1]]
+      ret.lateralTuning.pid.kpV, ret.lateralTuning.pid.kiV = [[0.15], [0.1]]
```

Comment:
seems like ki is too large wrt kp now. What of you use the sienna tuning?
```
ret.lateralTuning.pid.kpV, ret.lateralTuning.pid.kiV = [[0.3], [0.05]]
```
Would like to keep the tuning options limited. Also, anybody with this car that can confirm the tuning is better?

Reply:
I'll try those values and let you know how it goes.  I think I've tried Kp at 0.05 and it felt the same to me but I didn't look at cabana between the 2 values (0.1 vs 0.05).